components/Layout/Header/Header.jsx

```diff
@@ -1,7 +1,60 @@
-import { Header } from 'lbh-frontend-react';
+import Link from 'next/link';
+import { useContext } from 'react';
+import UserContext from 'components/UserContext/UserContext';
 
-const HeaderComponent = ({ serviceName }) => (
-  <Header serviceName={serviceName} homepageUrl="/" />
-);
+const HeaderComponent = ({ serviceName }) => {
+  const { user } = useContext(UserContext);
+  if (!user) {
+    return null;
```

Comment:
We need to render the header even if `user` is not defined, otherwise it won't render if you are not logged 😜